### DISCUSSION TITLE
Smythstoys test product not working

### DISCUSSION THREAD

**Discussion by kulab01** [jef/streetmerchant#1809] [Help]
I have tried to test Smythstoys with the template of the UK store and it doesn't seem to work.  The container seems to be correct and I have tried different one too without luck.

I used the example link:
https://www.smythstoys.com/uk/en-gb/video-games-and-tablets/video-games/call-of-duty-video-games/call-of-duty-black-ops-cold-war/call-of-duty-black-ops-cold-war-ps5/p/191951

I would appreciate if someone can also test it unless it is my installation.

**Comment by afst3r** [comment]
i would be interested in this too

**Comment by kulab01** [comment]
Anyone able to verify for me.  TIA.

**Comment by afst3r** [comment]
i have done the same thing, tried with fifa 21 its in stock for home and store and doesnt pick up the item

i have replaced the link to the ps5 and just insert this code

its a massive retail shop which would be great if it worked with

thanks

**Reply by breacher18** [reply]
I'm seeing this across all tests regardless of product or store. Has someone opened a bug report for this?

**Reply by breacher18** [reply]
Logged bug []here(https://github.com/jef/streetmerchant/issues/2096)
